docs/UsersGuide.asciidoc

```diff
@@ -973,6 +973,8 @@ remove assets accounted to that group:
   considered.
 * Assets which are also accounted to another group that has still space
   left are not considered.
+* Assets which are being updated will be kept ie. an asset that is
+  otherwise not accounted for but was modified on disk.
```

Comment:
Are assets still deleted but only that older assets are preferred for deletion? What happens if the job group quota is exceeded but all assets are "being updated"?

Also, instead of "ie." I guess you mean ", i.e.", right?

Reply:
I shouldn't have added it to the list, since in hindsight that's apparently confusing. This doesn't apply to assets which are in a job group.